src/__integ__/__snapshots__/themes.test.ts.snap

```diff
@@ -1594,6 +1596,7 @@ Object {
   "font-tabs-line-height": "22px",
   "font-tabs-size": "14px",
   "font-wayfinding-link-active-weight": "700",
+  "font-weight-bold": "700",
   "font-weight-heavy": "700",
```

Comment:
@DaemonCahill I understand `"font-weight-bold": "700"` has been added, but why `"font-weight-heavy": "700"` is still 700? Is this overwritten at some place?